src/platform/user/profile/vet360/components/base/VAPEditButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
This is based on the existing `Vet360ProfileFieldHeading` component, but is much more streamlined since it only needs to conditionally render an edit button